src/components/Discover/index.tsx

```diff
@@ -93,6 +104,37 @@ const Discover = () => {
         placeholder={<RequestCard.Placeholder />}
         emptyMessage={intl.formatMessage(messages.noRequests)}
       />
+      <div className="slider-header">
+        <Link href="/discover/watchlist">
+          <a className="slider-title">
+            <span>{intl.formatMessage(messages.plexwatchlist)}</span>
+            <ArrowCircleRightIcon />
+          </a>
+        </Link>
+      </div>
```

Comment:
This needs to be conditionally displayed along with the below slider. Otherwise, when there are no watchlist items, you just see the heading.

Reply:
Huh. Could have sworn this was in the below statement 🤔 Fixing!